examples/tutorial/flaskr/templates/base.html

```diff
@@ -1,9 +1,15 @@
 <!doctype html>
 <title>{% block title %}{% endblock %} - Flaskr</title>
 <link rel="stylesheet" href="{{ url_for('static', filename='style.css') }}">
+<script src="https://kit.fontawesome.com/a076d05399.js" crossorigin="anonymous"></script>
```

Comment:
apologies, the ticket didn't specify that we have our own font awesome kit, id 57e33e1f60.